config/production.js

```diff
@@ -14,7 +14,13 @@ module.exports = {
     sequelize: {
       // The `url` property is documented in sequelize-cli readme but not in Sequelize core
       url: process.env.DATABASE_URL,
-      logging: false
+      logging: false,
+      pool: {
+        max: 12,
+        min: 0,
+        acquire: 30000,
```

Comment:
these are current defaults, but im being more explicit by including them here. we can also modify them if we desire.